relayer/src/cli/programs/relay.ts

```diff
@@ -92,5 +92,18 @@ export class RelayerProgram {
         await this.l2RelayerTo.finalizeStep.execute(l1TransactionHash);
       }
     }
+
+    const relayParameters = await this.l2RelayerFrom.parseFillEventDataFromTxHash(
```

Comment:
Can we print a status message here before this additional step starts?
Something like: `Verifying successful relay operation...`

Would be nice for debugging purposes.